lib/helpers/utils.py

```diff
@@ -0,0 +1,7 @@
+from urllib3 import Retry
+
+
+def get_default_retries():
+    return Retry(
+        total=5, backoff_factor=0.1, status_forcelist=[500, 502, 503, 504]
+    )
```

Comment:
This is one of the rare places where a docstring for a simple function is warranted. What is backoff_factor? Why are only 500s retried instead of other status codes like `422`? Etc. 

The answers are obvious to intermediate and senior developers, but for junior devs who may very well use this library it's non-obvious and they may confuse even what a 500 means. They may think it is a local exception instead of a 500 on the checkr staging API, for example.

Reply:
I see. I'll add a docstring there. Thank you!